worksheet_A/TerminalHacking.cpp

```diff
@@ -41,6 +53,31 @@ int main()
 
 	// TODO: implement the rest of the game
 
+	std::string inputWord;
+	bool winGame = false;
+
+	std::cout << "The password is " << secret << "\n";
+
+	for (int numLives = 5; numLives > 0; numLives = numLives - 1) {
+		std::getline(std::cin, inputWord);
+		int output = getLikeness(inputWord, secret);
+
+		std::cout << "Your score is: " << output << "\n";
+
+
+
+		if (output == 5) {
+			std::cout << "You win!";
+			numLives = 0;
```

Comment:
Needs a comment saying that this breaks you out of the loop. Or you could modify the loop as follows, so that it ends if `winGame` becomes `true`:

```
for (int numLives = 5; numLives > 0 && !winGame; numLives = numLives - 1)
```

Or you could use the `break` keyword.